.github/workflows/error-ref-publisher.yaml

```diff
@@ -46,7 +46,7 @@ jobs:
 
       - name: Update and push docs
         run: |
-          echo '{ "errors_export": '$(cat ./helpers/errorutil_errors_export.json)' }' | jq > ./meshery/docs/_data/errorref/meshkit_errors_export.json
+          $ echo '{ "errors_export": "" }' | jq --slurpfile export ./helpers/errorutil_errors_export.json '.errors_export = $export[0]' > ./meshery/docs/_data/errorref/meshkit_errors_export.json
```

Comment:
Thanks @includestdlibrary. Just a very minor change and we'll be good to go : D
```suggestion
          echo '{ "errors_export": "" }' | jq --slurpfile export ./helpers/errorutil_errors_export.json '.errors_export = $export[0]' > ./meshery/docs/_data/errorref/meshkit_errors_export.json
```

Reply:
Ohh yess sorry 😅😅 I have not deleted that

Reply:
Gotcha :grin: